types/react-relay/index.d.ts

```diff
@@ -12,8 +12,9 @@
 //                 Janic Duplessis <https://github.com/janicduplessis>
 //                 Christian Ivicevic <https://github.com/ChristianIvicevic>
 //                 Marais Rossouw <https://github.com/maraisr>
+//                 Edvin Erikson <https://github.com/edvinerikson>
```

Comment:
@edvinerikson can you please confirm I've got this right, as well as just "agree" to have me add you here.

Reply:
Yep, this is right 👍🏼